src/nvim/popupmnu.c

```diff
@@ -756,3 +756,19 @@ int pum_get_height(void)
 {
   return pum_height;
 }
+
+/// Gets bounding of popup menu.
+dict_T *pum_get_bounding(void)
```

Comment:
"dimensions" or "properties" might be more applicable word than "bounding".

Reply:
There's a browser API called `getBoundingClientRect` that return similar properties for DOM, so how about `pum_get_bounding_rect`?